chart/env/dev.yaml

```diff
@@ -196,6 +196,7 @@ firstRows:
     limits:
       cpu: 1
       memory: "4Gi"
+  columnsMaxNumber: 1_000
```

Comment:
just fyi, if it's the same value as values.yaml, it's not necessary to repeat it. We can let it